plugins/woocommerce-blocks/tests/e2e/tests/templates/order-confirmation.block_theme.spec.ts

```diff
@@ -85,37 +85,4 @@ test.describe( 'Test the order confirmation template', async () => {
 				} )
 		).toBeVisible();
 	} );
-
-	test( 'Template can be modified', async ( {
```

Comment:
In removing this code, perhaps we can clean up removing line 10-14 and 25-27 as they're no longer used?

Reply:
Oh, good point, thanks! That file could be definitely cleaned up. Done in: 3eafde5f1dcabe601e2a5c1333d164d94adae38e.